tests/backup/backup_test.go

```diff
@@ -147,10 +147,32 @@ var _ = Describe("{BasicBackupCreateWithRules}", func() {
 				CreateBackupLocation(provider, backup_location_name, BackupLocationUID, CredName, CloudCredUID, bucketName, orgID)
 			}
 		})
+		Step("Creating backup schedule policies", func() {
+			dash.Info("Creating backup schedule policy of type interval")
+			interval_schedule_policy_info := CreateIntervalSchedulePolicy(5, int64(1*time.Minute), 2)
```

Comment:
In general, we aren't allowed to create schedule policy with time interval of less than 15 min,  Is the API allowing you to have 1 min interval polices? If yes then please file a bug for this and change it to 15 instead of one in `int64(1*time.Minute)`

Reply:
Changed it to 15 in automation 
Will recheck it again and then raise bug if required